app/models/course.rb

```diff
@@ -196,6 +195,14 @@ def word_count
     WordCount.from_characters(character_sum)
   end
 
+  def home_wiki
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)